main/tests/cypress/cypress/integration/tutorial/importing-data-into-openrefine.spec.js

```diff
@@ -0,0 +1,42 @@
+// This spec is implementation for chapter 02 of following tutorial https://librarycarpentry.org/lc-open-refine/02-importing-data/index.html
+
+describe(__filename, function () {
+  it('Create your first OpenRefine project (using provided data)', function () {
```

Comment:
This test doesn't contains any assertions, you might want to add a few checks to ensure the grid is as expected

Reply:
Sure👍

Reply:
done